var/spack/packages/openblas/package.py

```diff
@@ -0,0 +1,24 @@
+from spack import *
+
+class Openblas(Package):
+    """OpenBLAS: An optimized BLAS library"""
+    homepage = "http://www.openblas.net"
+    url      = "http://github.com/xianyi/OpenBLAS/archive/v0.2.15.tar.gz"
+
+    version('0.2.15', 'b1190f3d3471685f17cfd1ec1d252ac9')
+
+    # virtual dependency
+    provides('blas')
+    provides('lapack')
```

Comment:
Well I'm happy the concretization code is smarter than I thought it was :).  Ok I can merge this in.  I'll add a test for it later, to make sure we support providing multiple vdeps.  Thanks for all the contributions!